utils/webpack-utils.js

```diff
@@ -1,7 +1,13 @@
 function createExternals() {
   return [function(context, request, callback) {
-    if (request.indexOf('@playkit-js') === 0 || request.indexOf('preact') === 0) {
-      return callback(null, 'umd ' + request);
+    if (request.indexOf('@playkit-js') === 0) {
+      callback(null, 'umd ' + request);
+      return;
```

Comment:
Just making sore - this change is replacing the return type. Was this done intentionally